packages/rest/src/__tests__/integration/rest.application.integration.ts

```diff
@@ -163,6 +171,103 @@ describe('RestApplication (integration)', () => {
     await client.get(response.header.location).expect(200, 'Hi');
   });
 
+  context('mounting an Express router on a LoopBack application', async () => {
+    beforeEach('set up RestApplication', async () => {
+      givenApplication();
+      await restApp.start();
+      client = createRestAppClient(restApp);
+    });
+
+    it('gives precedence to an external route over a static route', async () => {
+      const router = express.Router();
+      router.get('/', function(_req: Request, res: Response) {
+        res.send('External dog');
+      });
+
+      restApp.static('/dogs', ASSETS);
```

Comment:
Do we silently replace the static route in this case?

Reply:
When it's looking for the route, it first checks if there's an external route, then if it can't find it then it uses the static one (see [here](https://github.com/strongloop/loopback-next/pull/2643/files#diff-4ac1e4b2069b2ff62309cfdd6b9dbe7bR103)), so I believe the static one still exists, but is overshadowed by the external one.